### PR TITLE
Enable write_riemann plugin and update to collectd-5.5.0

### PR DESCRIPTION
**Pull request by sharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # BOSH Release for collectd
 
-A super basic release for collectd
+A super basic release for collectd.  This fork renables the write_riemann plugin which
```
Comment (shinji62) [inline README.md:3]: No more a fork if it s merge..

File: jobs/collectd/templates/collectd.d/write_riemann.conf.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+LoadPlugin "write_riemann"
```
Comment (shinji62) [inline jobs/collectd/templates/collectd.d/write_riemann.conf.erb:1]: Maybe you should add if collectd.riemann_server empty so we don't load the plugin.

**Comment by poligraph** [comment]
Reasonable PR, waiting for merge from @shinji62

**Comment by drnic** [comment]
Sorry I didn't review the PRs when doing the simple bump of collectd to 5.7.1 and switching to `bosh2`. If you'd like to re-do the PR, great. Else perhaps I or someone else will take the good bits later.

**Comment by drnic** [comment]
Someone willing to cleanup/include the PR suggestions? /cc @poligraph @shinji62

**Comment by drnic** [comment]
This bosh release now has a CI that will automatically bump to new collectd releases https://ci.starkandwayne.com/teams/main/pipelines/collectd-boshrelease?groups=blobs&groups=collectd-boshrelease

It will also automatically build/compile new PRs.

**Pull request closed without merging**